dask_jobqueue/pbs.py

```diff
@@ -1,3 +1,5 @@
+from __future__ import division
+
```

Comment:
I use some division in slurm.py too. You took it in pbs.py, but not in slurm.py, is this intended?
I can see you've imported that also in the __init__.py file, I'm not sure what it does, and if it is enough for both pbs and slurm then (not coming from pythonic world).